source/js/model.js

```diff
@@ -390,6 +391,7 @@ function Game() {
     let _field = fieldSingleton.getInstance();
     let _winnerChecker = new WinnerChecker();
     let _iterator = new Iterator(getPlayersArray());
+    let movesCounter = 1;
```

Comment:
Please keep up with the coding style, use underscore before class-level variable names.